src/main/webapp/app/exercises/shared/feedback/collapse/feedback-collapse.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { faAngleDown, faAngleRight } from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+    selector: 'jhi-feedback-collapse',
+    styleUrls: ['./feedback-collapse.scss'],
+    templateUrl: './feedback-collapse.component.html',
+})
+/**
+ * smallCharacterLimit can be adjusted make smaller or bigger items collapsable
+ * isCollapsed tracks whether an item is currently open or closed
+ * text is any string passed to the component
+ */
+export class FeedbackCollapseComponent implements OnInit {
```

Comment:
Thats a leftover from the earlier implementation. Also, it isn't a real accordion but rather more like a truncated text https://css-tricks.com/multiline-truncated-text-with-show-more-button/.

I also noticed the accordion afterwards as a possibility for the display of groups in `feedback-node.component.ts` but decided against it since it would require me to change a lot about those components to fit the mock

Reply:
Tests fail due to missing coverage. I need to think of ways to write meaningful tests